suspendmanager.lua

```diff
@@ -531,3 +557,42 @@ end
 if not dfhack_flags.module then
     main({...})
 end
+
+-- Overlay Widget
+JobOverlay = defclass(JobOverlay, overlay.OverlayWidget)
+JobOverlay.ATTRS{
+    default_pos={x=-41,y=14},
+    default_enabled=true,
+    viewscreens='dwarfmode/ViewSheets/BUILDING',
+    frame={w=30, h=5},
+    frame_style=gui.MEDIUM_FRAME,
+    frame_background=gui.CLEAR_PEN,
+}
+
+function JobOverlay:init()
+    self:addviews{
+        widgets.Label{
+            frame={t=0, l=0},
+            text='Staying suspended:'
+        },
+        widgets.Label{
```

Comment:
can use a `widgets.WrappedLabel` for automatic wrapping, but if you do, you also need to force it to rewrap when the message changes. This means adding a bit of logic to detect when the message changes in `render`.

Reply:
not needed anymore (and only one line of real-estate)